modules/trusted-firmware-m/Kconfig

```
@@ -20,6 +20,13 @@ config TFM_BL2
 	bool
 	default n
 
+config TFM_SECURE_UART1
+	bool "Use UART1 for TF-M logging"
+	default y
+	help
```

should there be a:
```suggestion
	depends on !UART_1_NRF_UARTE
	help
```
here to ensure Secure uart1 cannot be enabled when uart1_nrf_uarte is enabled ?

---

Done